### ISSUE TITLE
Publish new version to clear warning "Failed propType: Invalid prop `destroyOnUnmount` of type `boolean`"

### ISSUE THREAD

**Issue by ShinyChang** [erikras/redux-form#232]
I update to v3.0.0 and got warning.

```
Warning: Failed propType: Invalid prop `destroyOnUnmount` of type `boolean` supplied to `ReduxFormConnector(IntjectIntl(Form))`, expected `function`. Check the render method of `ConnectedForm`.
```

And i checked code, it be fixed.

Could you publish new version?

**Comment by erikras** [comment]
Ha! It took me 30 seconds after I released v3 to fix that! Grr!! Will release v3.0.1.

**Comment by ShinyChang** [comment]
thank you @erikras
